java/ql/lib/semmle/code/java/security/CommandArguments.qll

```diff
@@ -155,7 +155,7 @@ private class CommandArgArrayImmutableFirst extends CommandArgumentArray {
   Expr getFirstElement() {
     result = this.getAWrite(0)
     or
-    not exists(this.getAWrite(0)) and
+    not exists(getAWrite(0)) and
```

Comment:
Is this compiler difference being tracked somewhere?

Reply:
I don't think there is an issue tracking it.  
But it has been mentioned to the core team, and I think `alexet` is looking into removing the pointless negative recursion, but that is apparently tricky (see the core channel from October 18th).